src/js/call.js

```diff
@@ -160,11 +160,11 @@ Call.asyncCreateTurnConfig = function(onSuccess, onError) {
     report.traceEventInstant('turn-config', config);
     setTimeout(onSuccess.bind(null, config), 0);
   } else {
-    Call.fetchCEODTurnConfig_(function(response) {
+    Call.fetchTurnConfig_(function(response) {
       var iceServer = {
-        'username': response.username,
-        'credential': response.password,
-        'urls': response.uris
+        'username': response.iceServers[0].username,
+        'credential': response.iceServers[0].credential,
+        'urls': response.iceServers[0].urls
       };
       var config = {'iceServers': [iceServer]};
```

Comment:
Use all provided ice servers: { 'iceServers': response.iceServers } instead of only the first on the list. I think it should work.